package io.hhplus.clean_arch.common

class CustomException (val type: CustomExceptionType): RuntimeException() { }